org.xverizex.nem-desktop.json

```
@@ -0,0 +1,41 @@
+{
+    "app-id" : "org.xverizex.nem-desktop",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "nem",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--socket=session-bus",
+        "--filesystem=home:create",
+        "--talk-name=org.xverizex.nem-desktop"
```

```suggestion
        "--filesystem=home:create"
```

this talk name is implied (it's the app id). As for the `:create` on `home`, I don't think so. Please restrict these permissions.